icu4c/source/test/intltest/compactdecimalformattest.cpp

```diff
@@ -188,9 +188,9 @@ static ExpectedResult kGermanCurrencyTestData[] = {
         {1.0, u8"1\\u00A0\\u20AC"},
         {12.0, u8"12\\u00A0\\u20AC"},
         {123.0, u8"120\\u00A0\\u20AC"},
-        {1234.0, u8"1200\\u00A0\\u20AC"},
-        {12345.0, u8"12.000\\u00A0\\u20AC"},
-        {123456.0, u8"120.000\\u00A0\\u20AC"},
+        {1234.0, u8"1,2\\u00A0Tsd.\\u00A0\\u20AC"},
```

Comment:
It could be a CLDR bug introduced by vetters, I fixed a couple of others. Markus should file an urgent CLDR bug and this can get fixed in CLDR before we release.